packaging/rpm/microshift.spec

```diff
@@ -261,6 +266,9 @@ systemctl enable --now --quiet openvswitch || true
 # Use Git command to generate the log and replace the VERSION string
 # LANG=C git log --date="format:%a %b %d %Y" --pretty="tformat:* %cd %an <%ae> VERSION%n- %s%n" packaging/rpm/microshift.spec
 %changelog
+* Fri Jan 27 2023 Miguel Angel Ajo Pelayo <majopela@rehat.com> 4.12.0
+- Add firewalld systemd service override configuration to avoid access to iptables NP-641
```

Comment:
I would not include NP-641 issue reference in the comment.

Reply:
It was standard practice in rpm packaging to include references to bugs, in fact at some point in history I remember you could not push a packaging change without the reference to a bz#  has this changed?

Reply:
The current history in the file does not include any issue references. If we decide to reference any issues, we should probably trace the history back and add it all along. For now, I would stick out.